performance_manager/lib/logging_utils.py

```diff
@@ -31,10 +32,14 @@ def add_metadata(self, **metadata: MdValues) -> None:
 
     def log_start(self) -> None:
         """log the start of a proccess"""
+        self.uuid = uuid.uuid4()
+        self.start_time = time.monotonic()
+        self.process_id = os.getpid()
```

Comment:
add process id as standard for easier multiprocessing debugging

Reply:
good idea.